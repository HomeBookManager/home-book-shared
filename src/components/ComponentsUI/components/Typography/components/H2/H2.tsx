import { FC } from 'react';

// components
import Typography from '../../Typography';

// types
import { TOmittedTypographyProps } from '../../types';
import { TypographyFontType } from '../../enums';

const H2: FC<TOmittedTypographyProps> = ({ children, ...restProps }) => (
  <Typography fontType={TypographyFontType.h2} {...restProps}>
    {children}
  </Typography>
);

export default H2;
