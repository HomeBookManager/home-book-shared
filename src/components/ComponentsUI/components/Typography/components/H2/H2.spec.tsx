import { render } from '@testing-library/react';

// components
import H2 from './H2';

describe('H2 snapshots', () => {
  it('should render H2', () => {
    //before
    const { asFragment } = render(<H2>H2</H2>);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
