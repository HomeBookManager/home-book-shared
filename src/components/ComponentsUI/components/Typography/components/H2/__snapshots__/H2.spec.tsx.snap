// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H2 snapshots should render H2 1`] = `
<DocumentFragment>
  <h2
    class="Typography Typography--regular Typography--normal Typography--h2 Typography--without-margin"
    data-test-text=""
    style="color: rgb(30, 40, 57);"
  >
    H2
  </h2>
</DocumentFragment>
`;
